[8.19] Add a known issue for Elastic Agents getting stuck in an `upgrade scheduled` state (#1841)
elastic/ingest-docs: docs/en/ingest-management/release-notes/release-notes-8.19.asciidoc

{fleet-server}::
* Upgrade golang.org/x/net to v0.34.0 and golang.org/x/crypto to v0.32.0. {fleet-server-pull}4405[#4405]

[discrete]
[[known-issues-8.19.0]]
=== Known issues

[[known-issue-2285-8.19.0]]
.{agents} remain in an "Upgrade scheduled" state
[%collapsible]
====

*Details* +

> There is a known issue where after a scheduled {agent} upgrade is cancelled, {agent} remains in an `Upgrade scheduled` state, and attempting to restart the upgrade on the UI returns an error: `The selected agent is not upgradeable: agent is already being upgraded.`.

Reviewer: ```suggestion
There is a known issue where an {agent} remains in an `Upgrade scheduled` state when a scheduled {agent} upgrade is cancelled. Attempting to restart the upgrade on the UI returns an error: `The selected agent is not upgradeable: agent is already being upgraded`.
```

Reviewer: Please consider this wording tweak to make the content easier for users to parse.  I trust your judgement.

Author: You’re right, it’s much better!
